### ISSUE TITLE
cc1: fatal error: ovis_json_lexer.c: No such file or directory

### ISSUE THREAD

**Issue by vsoch** [ovis-hpc/ldms#1260]
Hiya! I want to open this mostly for visibility - I was following the instructions [here](https://github.com/ovis-hpc/ovis-wiki/wiki/QuickStart-%28v4%29) and ran into this issue:

```console
vis_json_parser.y:58.1-7: warning: POSIX Yacc does not support %define [-Wyacc]
   58 | %define api.pure full
      | ^~~~~~~
make  all-recursive
make[5]: Entering directory '/ovis/lib/src/ovis_json'
make[6]: Entering directory '/ovis/lib/src/ovis_json'
  CC       ovis_json_test-ovis_json_test.o
  CC       ovis_json.lo
: -o ovis_json_lexer.c ovis_json_lexer.l
  CC       ovis_json_lexer.lo
cc1: fatal error: ovis_json_lexer.c: No such file or directory
```

Looking [at this file](https://github.com/ovis-hpc/ovis/blob/1e4c27fdc4fdd14dc1a94ebbd2954997fe017c99/lib/src/ovis_json/Makefile.am#L13) it looks like we need something called `$(LEX)` to generate that file. In retrospect I [see it here](https://github.com/ovis-hpc/ovis/blob/1e4c27fdc4fdd14dc1a94ebbd2954997fe017c99/lib/src/ovis_json/Makefile.am#L13) but I didn't see it [here](https://github.com/ovis-hpc/ovis-wiki/wiki/QuickStart-%28v4%29) so wanted to open an issue if others stumble on this they will easily find the solution. The solution is to:

```bash
$ apt-get install -y flex
```
And then it will build! And maybe the getting started guide could include that too. Thank you!

**Comment by morrone** [comment]
Yes, there was probably a warning the "configure" output, but AC_PROG_LEX doesn't error out when a lexer isn't found. We should probably add an autoconf test that checks if it is possible to execute "$LEX --version", and throw an error if it isn't.

**Comment by tom95858** [comment]
Please reopen if this is still an issue.

**Comment by morrone** [comment]
It look like addressed in #1394.
